src/components/card/Card.tsx

```diff
@@ -1,22 +1,38 @@
-import BookBody from '../../model/Book'
+import { useEffect, useState } from 'react'
+
+import { Book } from '../../model/Book'
 import './Card.css'
+import placeholderImg from '../../assets/placeholder/placeholderForBook.png'
 
-interface Book {
-  book: BookBody
+interface ShowBook {
+  book: Book
 }
 
-const Card = () => {
+const Card = ({ book } : ShowBook) => {
+  const [ placeholder , setPlaceholder ] = useState('')
+
+  useEffect(() => {
```

Comment:
Why do you need this state? Couldnt you just put placeholderImg in src?

Reply:
Yes i could. I fix.